app/src/main/java/org/torproject/android/ui/connect/ConnectFragment.kt

```diff
@@ -340,13 +340,13 @@ class ConnectFragment : Fragment(),
 
         binding.ivStatus.setImageResource(R.drawable.torstarting)
         val animHover = AnimationUtils.loadAnimation(context, R.anim.hover)
-        animHover.repeatCount = 7
+        animHover.repeatCount = 3
         animHover.repeatMode = Animation.REVERSE
         binding.ivStatus.animation = animHover
         animHover.start()
 
         val animShadow = AnimationUtils.loadAnimation(context, R.anim.shadow)
-        animShadow.repeatCount = 7
```

Comment:
wondering if these assignments are necessary or if this can all be done in XML. Same with the `repeatMode` assignment too. I'm in favor of this PR and can merge, but would like to get rid of this kotlin if it's unnecessary